src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-sync-data-info/rgw-sync-data-info.component.html

```diff
@@ -18,7 +18,7 @@
 </ng-template>
 <ul class="me-2">
   <ng-template #upToDateTpl>
-    <li class="badge badge-success">Up to Date</li>
+    <cds-tag class="tags-success">Up to Date</cds-tag>
```

Comment:
why removing `<li>` here and not below ?

Reply:
under <li> added cds-tag